core-plugin/testSrc/com/google/gct/idea/debugger/ui/CloudDebugHistoricalSnapshotsTest.java

```diff
@@ -61,17 +28,10 @@
 import org.junit.Test;
 import org.mockito.Mockito;
 
-import java.awt.Component;
-import java.awt.Dimension;
-import java.awt.event.InputEvent;
 import java.util.ArrayList;
-import java.util.Comparator;
 import java.util.List;
-import java.util.Set;
 
-import javax.swing.Icon;
 import javax.swing.JComponent;
-import javax.swing.event.HyperlinkListener;
 
 
 // TODO: This test hangs when run as part of the full suite.
```

Comment:
Still broken, so leaving the ignore in. I think we need to use IdeaTestCase or one of the other built ins so they all the state is managed correctly.